packages/react-hooks/src/useThrottledCallback.ts

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useMemo, useRef } from 'react';
+import type { DebouncedFunc, ThrottleSettings } from 'lodash';
+import throttle from 'lodash.throttle';
+
+/**
+ * Wraps a given callback in a cancelable, throttled function. The throttled
+ * callback is automatically cancelled if the callback reference changes or the
```

Comment:
Also of note the `useDebouncedCallback` should probably be structured similarly, as right now it could continually cancel if the callback passed in changes and then never gets called.